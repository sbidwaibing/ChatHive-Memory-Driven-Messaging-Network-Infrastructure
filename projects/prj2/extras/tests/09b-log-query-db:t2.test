 ? Sysprog
.
? Sysprog 100
.
? Sysprog 2
.
? Sysprog 100 #ruby
.
? Sysprog 100 #unix #pipe
.
? SysProg 100 #db #mongoDB #json #js #python #ruby #js #db
.
? SysProg 100  #unix #db
.
