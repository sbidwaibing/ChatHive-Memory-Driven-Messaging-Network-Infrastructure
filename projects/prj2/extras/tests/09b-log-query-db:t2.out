ok
YYYY-MM-DDThh:mm:ss.ttt
@jane sysprog #file-descriptor #pipe #syscall #unix
The pipe() syscall returns two file descriptors fd[2] to a 
pipe such that fd[0] refers to the read end of the pipe 
and fd[1] refers to the write end of the pipe. This pipe 
is typically used for inter-process communication.
ok
YYYY-MM-DDThh:mm:ss.ttt
@jane sysprog #file-descriptor #pipe #syscall #unix
The pipe() syscall returns two file descriptors fd[2] to a 
pipe such that fd[0] refers to the read end of the pipe 
and fd[1] refers to the write end of the pipe. This pipe 
is typically used for inter-process communication.
YYYY-MM-DDThh:mm:ss.ttt
@tom sysprog #fork #stdio #syscall #unix
Worth noting: If stdio buffers are not flushed before a fork(), 
then the child will also get a copy of that buffer.
YYYY-MM-DDThh:mm:ss.ttt
@zdu sysprog #db #js #json #mongodb #python #ruby
mongodb supports storing structured documents (essentially 
JSON) and can be accessed from many programming languages
include popular scripting languges like JavaScript, Python 
and Ruby.
YYYY-MM-DDThh:mm:ss.ttt
@zdu sysprog #db #sqlite
sqlite is pretty cool
ok
YYYY-MM-DDThh:mm:ss.ttt
@jane sysprog #file-descriptor #pipe #syscall #unix
The pipe() syscall returns two file descriptors fd[2] to a 
pipe such that fd[0] refers to the read end of the pipe 
and fd[1] refers to the write end of the pipe. This pipe 
is typically used for inter-process communication.
YYYY-MM-DDThh:mm:ss.ttt
@tom sysprog #fork #stdio #syscall #unix
Worth noting: If stdio buffers are not flushed before a fork(), 
then the child will also get a copy of that buffer.
ok
YYYY-MM-DDThh:mm:ss.ttt
@zdu sysprog #db #js #json #mongodb #python #ruby
mongodb supports storing structured documents (essentially 
JSON) and can be accessed from many programming languages
include popular scripting languges like JavaScript, Python 
and Ruby.
ok
YYYY-MM-DDThh:mm:ss.ttt
@jane sysprog #file-descriptor #pipe #syscall #unix
The pipe() syscall returns two file descriptors fd[2] to a 
pipe such that fd[0] refers to the read end of the pipe 
and fd[1] refers to the write end of the pipe. This pipe 
is typically used for inter-process communication.
ok
YYYY-MM-DDThh:mm:ss.ttt
@zdu sysprog #db #js #json #mongodb #python #ruby
mongodb supports storing structured documents (essentially 
JSON) and can be accessed from many programming languages
include popular scripting languges like JavaScript, Python 
and Ruby.
ok
